chapters/annotations.tex

```diff
@@ -518,17 +526,18 @@ \subsection{Single Use of Class}\label{annotation-for-single-use-of-class}\label
 For state machines it is useful to have single instances of local classes.
 This can be done using:
 \begin{lstlisting}[language=modelica]
-annotation(singleInstance = true)
+/*literal*/ constant Boolean singleInstance;
 \end{lstlisting}
 
-The annotation \fmtannotationindex{singleInstance} in a class indicates that there should only be one component instance of the class, and it should be in the same scope as the class is defined.
+The annotation \fmtannotationindex{singleInstance} if \lstinline!true! in a class indicates that there should only be one component instance of the class, and it should be in the same scope as the class is defined.
 The intent is to remove the class when the component is removed and to prevent duplication of the component.
+If \lstinline!false! it has no impact.
 
 \subsection{Connection Restrictions}\label{connection-restrictions}
 
 A connector component declaration may have the following annotation:
 \begin{lstlisting}[language=modelica]
-annotation(mustBeConnected = "message");
+/*literal*/ constant String mustBeConnected = "$\mathit{message}$";
```

Comment:
Shouldn't we align this with `singleInstance`, so that we only use declaration equations for specifying defaults?

Reply:
Good catch.

Reply:
Has been corrected.